### ISSUE TITLE
seriesBarsPlugin() fails if there is only one group

### ISSUE THREAD

**Issue by maxhq** [leeoniya/uPlot#449]
The `seriesBarsPlugin()` (https://github.com/leeoniya/uPlot/blob/master/demos/multi-bars.html) fails if there is only one group, e.g. in the example if the data is reduced to the uPlot lib.

It can be easily seen when directly passing the calculated data objects from the example to uPlot:
```javascript
// works:
let data = [[1,2],[38,954],[146,208],[12.8,13.3],[3,4.7],[902,4123],[112,165]];
// fails:
let data = [[1],  [38],    [146],    [12.8],     [3],    [902],     [112]];

const opts = {
  width: 1920,
  height: 600,
  title: "Line Charts (166,650 points)",
  axes: [
    {},
    { show: false }
  ],
  legend: { width: 0, live: false, },
  padding: [ null, 0, null, 0 ],
  series: [
    { label: "Lib Name" },
    { label: "Lib Size (KB)", fill: "#33BB55", scale: "size"},
    { label: "Render Time (ms)", fill: "#B56FAB", scale: "rend"},
    { label: "Peak Heap (MB)", fill: "#BB1133", scale: "mem"},
    { label: "Final Heap (MB)", fill: "#F79420", scale: "mem"},
    { label: "Interact 10s (ms)", fill: "#558AB5", scale: "inter"},
    { label: "Toggle 6x (ms)", fill: "#FAD55C", scale: "toggle"},
  ],
  plugins: [
    seriesBarsPlugin({
        labels: () => ['uPlot', 'ECharts'],
        dir: 1,
        ori: 0,
    }),
  ]
};

let u = new uPlot(opts, data, document.body);
```

**Comment by maxhq** [comment]
... this affects version 1.6.3. I have to check if it's fixed in 1.6.4

**Comment by leeoniya** [comment]
yep, this got fixed in 1.6.4: https://github.com/leeoniya/uPlot/commit/7cc18d7422001122c54d765923f2d4a0e9ab4f7e

(the isNaN and Infinity test)
